Add a `jose-cose` section to implementer info.

BigBlueHat opened the pull request:
Data and format are from
https://github.com/w3c/vc-jose-cose-test-suite/commit/a37a75ee391ceb9b121629a816eff712c91beb20

Prepares for https://github.com/w3c/vc-jose-cose-test-suite/pull/30

BigBlueHat committed:
Add a `jose-cose` section to implementer info.

Data and format are from
https://github.com/w3c/vc-jose-cose-test-suite/commit/a37a75ee391ceb9b121629a816eff712c91beb20

davidlehn reviewed (approved):
Approving to keep things moving.
- These implementation files *really* need optional URLs for both the vendor and implementation.  Maybe other metadata too.
- It's not clear what "Independent" means.  Is that a company, project, meta name, or something else?
- I'm guessing the diff failure is due to not handling new files and can be handled later.

BigBlueHat committed:
Rename independent implementation to decentralgabe.

Matches the commit log at
https://github.com/w3c/vc-jose-cose-test-suite/tree/main/implementations/independent
